### PR TITLE
dev-libs/granite: become proxied maintainer; bump to version 5.5.0

### PR DESCRIPTION
**Pull request by xgqt**
Package-Manager: Portage-3.0.12, Repoman-3.0.2
Signed-off-by: Maciej Barć <xgqt@protonmail.com>

### CODE REVIEW

File: dev-libs/granite/granite-5.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			"lib/Widgets/OverlayBar.vala"
+			"lib/Widgets/SeekBar.vala"
+			"lib/Widgets/StorageBar.vala"
+			"lib/Widgets/Toast.vala"
+			"lib/Widgets/Welcome.vala"
+		)
+		for src_file in "${doc_sed_list[@]}"; do
+			sed -ie "s@{{../doc@{{${BUILD_DIR}/doc@g" \
+				"./${src_file}" || die "Failed to fix docs for ./${src_file}"
+		done
```
Comment (juippis) [inline dev-libs/granite/granite-5.5.0.ebuild:60]: Could this be replaced with a simple `find -exec sed` line?

File: dev-libs/granite/granite-5.5.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+inherit meson vala xdg
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc +introspection"
```
Comment (juippis) [inline dev-libs/granite/granite-5.5.0.ebuild:18]: I don't understand what this USE flag does. How does granite change depending on the `deps[introspection]` state?

Comment (xgqt) [reply dev-libs/granite/granite-5.5.0.ebuild:18]: @juippis well... `option('introspection', type: 'boolean', value: true, description: 'Whether to build introspection files')`

File: dev-libs/granite/granite-5.5.0.ebuild (hunk 1 above)
Comment (xgqt) [reply dev-libs/granite/granite-5.5.0.ebuild:60]: > Could this be replaced with a simple `find -exec sed` line?

will do

File: dev-libs/granite/granite-5.5.0.ebuild (hunk 2 above)
Comment (juippis) [reply dev-libs/granite/granite-5.5.0.ebuild:18]: But you don't have it defined in `src_configure` :)

Comment (xgqt) [reply dev-libs/granite/granite-5.5.0.ebuild:18]: @juippis Blame the last maintainer. >:-)

Comment (xgqt) [reply dev-libs/granite/granite-5.5.0.ebuild:18]: @juippis Oh! Optional introspection support is only on master!

**Review by juippis — APPROVED** [review]

**Comment by xgqt** [comment]
Thanks for cooperation.

**Pull request closed without merging**